apps/pie-docs/src/components/checkbox/motion/motion.md

```diff
@@ -0,0 +1,86 @@
+---
+eleventyNavigation:
+    key: Motion
+    parent: 'Checkbox'
+    order: 2
+shouldShowContents: true
+eleventyComputed:
+    hover: "{% include './hover.json' %}"
+    active: "{% include './active.json' %}"
+    selected: "{% include './selected.json' %}"
+    unselected: "{% include './unselected.json' %}"
+---
+
+## Example
+
+Illustrates the animations that happen in the checkbox component. In this component we have animations on hover and active states as well as when the component is selected and unselected.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/checkbox/motion-example.svg",
+    alt: "Three checkboxes in a horizontal row. First one in default state, second one in hover state and third one in checked state.",
+    width: "586px",
+    height: "m"
+} %}
+
+---
+
+## Hover
+
+When the user hovers over an unselected checkbox (on:Hover), the hover state $hover01 fades in at 200ms. When a user moves the mouse away (off:Hover) the hover state $hover01 fades out at 200ms.
+
+{% contentWrapper %}
+    {% contentPageImage {
+        src:"../../../assets/img/components/checkbox/checkbox-hover-animation.svg",
+        alt: "A chart showing timing in milliseconds for hover 01 property."
+    } %}
+    {% componentDetailsTable { tableData: hover } %}
+{% endcontentWrapper %}
+
+---
+
+## Active
+
+When the user clicks a checkbox (on:Click), the active state $active01 fades in at 100ms and then out at 100ms.
```

Comment:
can we wrap any code-style copy in `` for better visual clarity? So in this case `$active01`